engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/DevProcessConfigCreator.scala

```diff
@@ -532,12 +533,16 @@ class SimpleTypesService extends Service with Serializable {
              rawParam: String,
 
              @ParamName("intParam")
-             @RawEditor intParam: Int,
+             @RawEditor
```

Comment:
Why you've used `RawEditor` for this? Raw editor is for expressions, IMO better option would be to pick simple string editor...

Reply:
I've used it only for manual FE tests. In fact this param has already existed before my changes..

Reply:
I know purpose of this class :) It is "added value" for all devs. I just have in mind that it would be nice to change this editor to better present ability of our editors/validators